.github/ISSUE_TEMPLATE/graduation.md

```diff
@@ -0,0 +1,387 @@
+
+- [x] **Document complete list of current maintainers, including names, contact information, domain of responsibility, and affiliation.**
+
+<!-- (Project assertion goes here) --> 
+
+[Maintainers](https://github.com/kyverno/kyverno/blob/main/MAINTAINERS.md).
+
+- [x] **A number of active maintainers which is appropriate to the size and scope of the project.**
+
+<!-- (Project assertion goes here) --> 
+
+The Kyverno project currently has eight (8) maintainers from four (4) affiliations.
+
+- [x] **Document a complete maintainer lifecycle process (including roles, onboarding, offboarding, and emeritus status).**
+
+<!-- (Project assertion goes here) --> 
+
+This is documented under [Maintainers](https://kyverno.io/community/#maintainers) section.
+
+- [x] **Demonstrate usage of the maintainer lifecycle with outcomes, either through the addition or replacement of maintainers as project events have required.**
+
+<!-- (Project assertion goes here) --> 
+
+The process of onboarding a maintainer is initiated by submitting a pull request. 
+
+For example, [feat: add myself (vishal-chdhry) to maintainers list #9125](https://github.com/kyverno/kyverno/pull/9125).
+
+The same with offboarding process, and emeritus maintainers are listed at: 
+
+https://github.com/kyverno/kyverno/blob/main/MAINTAINERS.md#maintainers-emeritus
+
+
+- [x] **Project maintainers from at least 2 organizations that demonstrates survivability.**
+
+<!-- (Project assertion goes here) --> 
+
+Currently maintainers are from four (4) organizations including Nirmata, Stackwatch, Ohio Supercomputer Center, Giant Swarm GmbH.
+
+- [ ] **Code and Doc ownership in Github and elsewhere matches documented governance roles.**   
+    - [x] [CODEOWNERS](https://github.com/kyverno/kyverno/blob/main/CODEOWNERS)
+    - [ ] doc owners?
+
+<!-- (Project assertion goes here) --> 
+
+- [x] **Document agreement that project will adopt CNCF Code of Conduct.**
+
+<!-- (Project assertion goes here) --> 
+
+Kyverno follows the [Code of Conduct](https://github.com/kyverno/kyverno/blob/main/CODE_OF_CONDUCT.md), which is aligned with the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
+
+
+- [x] **CNCF Code of Conduct is cross-linked from other governance documents.**
+
+<!-- (Project assertion goes here) -->
+
+This is documented on Github [Code of Conduct](https://github.com/kyverno/kyverno/blob/main/GOVERNANCE.md#code-of-conduct) and Kyverno website under Project Governance, see [here](https://kyverno.io/community/#project-governance).
+
+- [ ] **All subprojects, if any, are listed.**
+
+<!-- (Project assertion goes here) --> 
+
+
+- [x] **If the project has subprojects: subproject leadership, contribution, maturity status documented, including add/remove process.**
+
+<!-- (Project assertion goes here) --> 
+
+This is clarified in the [Project Governance section](https://main.kyverno.io/community/#project-governance).
```

Comment:
```suggestion
This is clarified in the [Project Governance section](https://kyverno.io/community/#project-governance).
```

Reply:
The project governance has been moved to `kyverno/community` GitHub repo, updated to https://github.com/kyverno/community/blob/main/GOVERNANCE.md.